From b8923e0f9ef28ac5e620196951f381a559a7137e Mon Sep 17 00:00:00 2001
From: Sean Busbey <busbey@apache.org>
Date: Tue, 29 Sep 2015 20:00:49 -0500
Subject: [PATCH 104/410] HBASE-14515 make sure spark tests are skippable like
 the other modules.

Conflicts:
	hbase-spark/pom.xml

Change-Id: I99ce5721738192cd3dfe8f123ecc9ae6eb0f8a05
Reason: New Feature
Author: Sean Busbey
Ref: CDH-35578
---
 hbase-spark/pom.xml |   16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/hbase-spark/pom.xml b/hbase-spark/pom.xml
index e0f84b1..068f93a 100644
--- a/hbase-spark/pom.xml
+++ b/hbase-spark/pom.xml
@@ -589,4 +589,20 @@
         </plugins>
     </build>
 
+  <profiles>
+    <!-- Skip the tests in this module -->
+    <profile>
+      <id>skipSparkTests</id>
+      <activation>
+        <property>
+          <name>skipSparkTests</name>
+        </property>
+      </activation>
+      <properties>
+        <surefire.skipFirstPart>true</surefire.skipFirstPart>
+        <surefire.skipSecondPart>true</surefire.skipSecondPart>
+      </properties>
+    </profile>
+  </profiles>
+
 </project>
-- 
1.7.9.5

